app/dashboard/static/js/app/view-tests-job-branch-kernel-plan.2020.1.js

```diff
@@ -0,0 +1,597 @@
+            .done(getRunsDone);
+    }
+
+    function testCountFailed() {
+        /* The chart will not be shown */
+    }
+
+    function testCountDone(response) {
+        updateChart(response);
+    }
+
+    function getTestCount() {
+        var qStr;
+        var batchOps;
+        var deferred;
+
+        qStr = URI.buildQuery({
+            'job': gJob,
+            'kernel':  gKernel,
+            'git_branch':  gBranch,
+            'plan':  gPlan,
+        });
+
+        batchOps = []
+
+        batchOps.push({
+            method: 'GET',
+            operation_id: 'test-total-count',
+            resource: 'count',
+            document: 'test_case',
+            query: qStr,
+        });
+
+        batchOps.push({
+            method: 'GET',
+            operation_id: 'test-success-count',
+            resource: 'count',
+            document: 'test_case',
+            query: qStr + '&status=PASS',
+        });
+
+        batchOps.push({
+            method: 'GET',
+            operation_id: 'test-fail-count',
+            resource: 'count',
+            document: 'test_regression',
+            query: qStr,
+        });
+
+        batchOps.push({
+            method: 'GET',
+            operation_id: 'test-unknown-count',
+            resource: 'count',
+            document: 'test_case',
+            query: qStr + '&status=FAIL&status=SKIP&regression_id=null',
+        });
+
+        deferred = request.post(
+            '/_ajax/batch', JSON.stringify({batch: batchOps}));
+
+        $.when(deferred)
+            .fail(error.error, testCountFailed)
+            .done(testCountDone);
+    }
+
+    if (document.getElementById('job-name') !== null) {
+        gJob = document.getElementById('job-name').value;
+    }
+    if (document.getElementById('branch-name') !== null) {
+        gBranch = document.getElementById('branch-name').value;
+    }
+    if (document.getElementById('kernel-name') !== null) {
+        gKernel = document.getElementById('kernel-name').value;
+    }
+    if (document.getElementById('plan-name') !== null) {
+        gPlan = document.getElementById('plan-name').value;
+    }
+    if (document.getElementById('file-server') !== null) {
+        gFileServer = document.getElementById('file-server').value;
+    }
+
+    gResultFilter = filter('data-filter');
+
+    Array.prototype.forEach.call(
+        document.querySelectorAll('.btn-group > .btn'),
+        function(btn) {
+            btn.addEventListener('click', function() {
+                Array.prototype.forEach.call(
+                    btn.parentElement.children, function(element) {
+                    if (element === btn) {
+                        html.addClass(element, 'active');
+                    } else {
+                        html.removeClass(element, 'active');
+                    }
+                });
+            });
+        }
+    );
+
+    setTimeout(getRuns, 10);
+    setTimeout(getTestCount(), 10);
```

Comment:
unnecessary parentheses in `getTestCount`

Reply:
Fixed.